Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="125" judge-id="41" status="SE" scoring="ACM" run-tests="1" contest-id="3205" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>6971f21c-d2ea-4598-b346-e35000f23cde</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="286" real-time="502" max-memory-used="18698240" >
  <program-stats-str>{ utime=252, stime=34, ptime=286, rtime=502, maxvsz=18698240, maxrss=19144704, nvcsw=2, nivcsw=36 }</program-stats-str>
      <input size="10">1 2 3 4 5
</input>
      <output size="6">1
3
5
</output>
      <stderr size="278">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000125&quot;, line 7, in &lt;module&gt;
    print (a[b])
IndexError: list index out of range
</stderr>
    </test>
  </tests>
</testing-report>
